pkg/transform/kubernetes/kubernetes.go

```diff
@@ -22,17 +22,17 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/davecgh/go-spew/spew"
 	"github.com/kedgeproject/kedge/pkg/spec"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/davecgh/go-spew/spew"
 	"github.com/pkg/errors"
 	"k8s.io/client-go/pkg/api"
 	"k8s.io/client-go/pkg/api/resource"
-	"k8s.io/client-go/pkg/runtime"
-	"k8s.io/client-go/pkg/util/intstr"
-
-	log "github.com/Sirupsen/logrus"
 	api_v1 "k8s.io/client-go/pkg/api/v1"
 	ext_v1beta1 "k8s.io/client-go/pkg/apis/extensions/v1beta1"
+	"k8s.io/client-go/pkg/runtime"
+	"k8s.io/client-go/pkg/util/intstr"
```

Comment:
this happened according to this comment https://github.com/kedgeproject/kedge/pull/149#issuecomment-315051859